tests/integration/span_events/test_span_events_are_created_upon_caught_exception.php5.php

```diff
@@ -9,6 +9,13 @@
 when an exception is handled by the exception handler.
```

Comment:
Actually, do we need this file anymore, can we delete it? The name is misleading because it says php5, but, as written, will run on PHP 7 as well.
It also looks like a duplicate of [tests/integration/span_events/test_span_events_are_created_upon_caught_exception.php7.php](https://github.com/newrelic/newrelic-php-agent/pull/748/files#diff-6346f826d98c0bba03ab77c273fc53aaa637f342e7c865818a19afda58a1fefc)

Reply:
You're right! Support for PHPs 5.5 and 5.6 ended with [10.11.0.3](https://github.com/newrelic/newrelic-php-agent/releases/tag/v10.11.0.3) release. I'll remove it.
P.S. It is not duplicate of the `php7` version of this test, because `php7` version checks for code level metrics attributes.

Reply:
`php5` version of the test removed in 91aa2e29a7699960f9e36df096e152e31c14f3e1.